docs/community/faqs.md

```diff
@@ -7,7 +7,7 @@ keywords:
 ---
 
 1. **How do I find good first issues?**
- we recommend checking out sites like [good first issues](https://goodfirstissues.com/) and [first timers only](https://www.firsttimersonly.com/). they will lead you to projects and issues worth working on.  we also recommend using the `good first issue` label in the search engine of the project you want to contribute to. check out [this youtube short from github](https://www.youtube.com/watch?v=k3l_rurkoxg) to learn more. furthermore, check out our daily "who's looking for open source contributors?" on [dev.to](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
+ we recommend checking out sites like [good first issues](https://goodfirstissues.com/) and [first timers only](https://www.firsttimersonly.com/). they will lead you to projects and issues worth working on.  we also recommend using the `good first issue` label in the search engine of the project you want to contribute to. check out [this guide from freeCodeCamp](https://www.freecodecamp.org/news/how-to-find-good-first-issues-on-github/) to learn more. furthermore, check out our daily "who's looking for open source contributors?" on [dev.to](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
```

Comment:
```suggestion
 We recommend checking out sites like [good first issues](https://goodfirstissues.com/) and [first timers only](https://www.firsttimersonly.com/). they will lead you to projects and issues worth working on.  We also recommend using the `good first issue` label in the search engine of the project you want to contribute to. Check  out [this guide by our Triage Team member, Christine Belzie,on freeCodeCamp](https://www.freecodecamp.org/news/how-to-find-good-first-issues-on-github/) to learn more. Furthermore, check out our daily "who's looking for open source contributors?" on [dev.to](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
```

Reply:
@CBID2, I had planned to create an issue addressing the need for proper capitalization after periods. Do you think it not necessary, or should I proceed with making the necessary fix within this pull request?

Reply:
> @CBID2, I had planned to create an issue addressing the need for proper capitalization after periods. Do you think it’s not necessary, or should I proceed with making the necessary fix within this PR?

Personally, I think it would be best to do these revisions on a separate PR @RitaDee.